### ISSUE TITLE
Continually “Awaiting credentials”

### ISSUE THREAD

**Issue by GHN1013** [brbeaird/SmartThings-MyQ-Edge#10]
Hi there,

was able to follow through all steps to install and able to see new MyQ Controller device on ST app, however after entering in username and PW, nothing happens. Keep seeing the “Awaiting credentials”.  I triple checked my email and PW are correct.

**Comment by rgagnon1967** [comment]
Same here. I am using a Google account to log in to the MyQ app on my phone, so this may be an issue. If so, Google account support would be very much appreciated, otherwise I will have to create another MyQ account and transfer my devices there.

**Comment by rgagnon1967** [comment]
Confirmed lack of support for Google accounts (log from bridge server):
myQ API: Invalid myQ credentials given. Check your login and password.

**Comment by GHN1013** [comment]
Interestingly, I’ve had no issues logging into MyQ using Google email on original GitHub MyQ Lite on SmartThings IDE.

**Comment by AdventureJon** [comment]
I had the same issue and solved it by following the "Forgot Password" process on MyQ account.   This allowed me to set a native password for MyQ instead of using Google Authentication.   This solved the "Awaiting credentials" problem and made it to the next step.

**Comment by rgagnon1967** [comment]
@AdventureJon I confirm. I did the same as you and it fixed the issue.

**Comment by brbeaird** [comment]
This still seems really odd to me. The "Awaiting credentials" status should just mean the Edge driver does not think it has credentials to use from the MyQ-Connector device and is thus just waiting to do anything. In theory, deleting username/password out of settings, then re-entering them should trigger it to try connecting again. I would have to see more logs to figure out why this happens.

**Comment by GHN1013** [comment]
Yeah it was odd indeed.  Originally, after entering correct Username/PW, it would literally just hang.  Thanks for your support.
